src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/CitizenHappinessHandler.java

```diff
@@ -64,14 +64,14 @@ public CitizenHappinessHandler(final ICitizenData data)
           new Tuple[] {new Tuple<>(IDLE_AT_JOB_COMPLAINS_DAYS, 0.5), new Tuple<>(IDLE_AT_JOB_DEMANDS_DAYS, 0.1)}));
 
         add(new StaticHappinessModifier(SCHOOL, 1.0, () -> data.isChild() ? data.getJob() instanceof JobPupil ? 2 : 0 : 1));
-        add(new StaticHappinessModifier(SECURITY, 5.0, () -> getGuardFactor(data.getColony())));
+        add(new StaticHappinessModifier(SECURITY, 4.0, () -> getGuardFactor(data.getColony())));
         add(new StaticHappinessModifier(SOCIAL, 2.0, () -> getSocialModifier(data.getColony())));
-        add(new StaticHappinessModifier(SATURATION, 1.0, () -> data.getSaturation() / 10.0));
+        add(new StaticHappinessModifier(SATURATION, 2.0, () -> (data.getSaturation() + 5.0) / 10.0));
 
-        add(new ExpirationBasedHappinessModifier(DAMAGE, 1.0, () -> 0.0, 1));
-        add(new ExpirationBasedHappinessModifier(DEATH, 2.0, () -> 0.0, 3));
+        add(new ExpirationBasedHappinessModifier(DAMAGE, 2.0, () -> 0.0, 1));
+        add(new ExpirationBasedHappinessModifier(DEATH, 3.0, () -> 0.0, 3));
         add(new ExpirationBasedHappinessModifier(RAIDWITHOUTDEATH, 1.0, () -> 2.0, 3));
-        add(new ExpirationBasedHappinessModifier(SLEPTTONIGHT, 2.0, () -> data.getJob() instanceof AbstractJobGuard ? 1 : 0.0, 3, true));
+        add(new ExpirationBasedHappinessModifier(SLEPTTONIGHT, 1.0, () -> data.getJob() instanceof AbstractJobGuard ? 1 : 0.0, 3, true));
     }
```

Comment:
Nah, this makes it too easy. Security and sleeping should stay high as they are. The rest is ok

Reply:
I decreased sleeping because, on servers with Morpheus, colonists rarely get a full night's sleep. I decreased security but increased dmg and death to compensate.

Reply:
Yes, and it's great how players try to make sure in teams that the needs are met. That's on purpose.

Reply:
Players will also help out new players by skipping the night. Can we set it to 1.5? Is that possible?

Reply:
The current value is fine, in doesn't bring down happyiness so badly at all. I balanced the current settings in a way that you can get full happiness even without it.